src/Tests/AuthoringConsumptionTest/AuthoringConsumptionTest.vcxproj

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\..\packages\Microsoft.WinUI.3.0.0-preview3.201113.0\build\native\Microsoft.WinUI.props" Condition="Exists('..\..\packages\Microsoft.WinUI.3.0.0-preview3.201113.0\build\native\Microsoft.WinUI.props')" />
+  <Import Project="..\..\packages\Microsoft.WinUI.3.0.0-zmain.210514.0\build\native\Microsoft.WinUI.props" Condition="Exists('..\..\packages\Microsoft.WinUI.3.0.0-zmain.210514.0\build\native\Microsoft.WinUI.props')" />

```

Comment:
Ideally we do not want to use internal versions to allow others externally to be able to build this repo.  Is there any version still on the public feed that we can use instead?  Otherwise we would need to move to reunion.